src/interface/projectData.ts

```diff
@@ -0,0 +1,13 @@
+export default interface DataType {
+  id: number;
+  projectName: string;
+  projectDescription: string;
+  imageURL: string;
+  createrDescription: string;
+  projectUrl: string;
+  githubRepoURL: string;
+  githubURL: string;
+  githubProfileURL: string;
+  createrName: string;
+  categories: any[];
+}
```

Comment:
public/data/project.json에 color가 추가된것 같아 interface 수정이 필요해 보입니다!

{
    "id": "0",
    "projectName": "Hello, GSM",
    "projectDescription": "Hello, GSM은 광주소프트웨어마이스터고등학교 입학 지원시스템입니다.",
    "imageURL": "/images/Hello,gsm.png",
    "createrDescription": "https://themoment-team.notion.site/2b3875e62b2a4002beb9db71c85db262",
    "projectUrl": "https://www.hellogsm.kr",
    "githubRepoURL": ["https://github.com/themoment-team/hello-gsm-about"],
    "githubURL": "https://github.com/themoment-team",
    "githubProfileURL": "https://github.com/themoment-team.png",
    "createrName": "더모먼트",
    "categories": ["#Next.js", "#Nest.js", "#React"],
    "color": "#7090B866"
  },



```suggestion
export default interface DataType {
  id: number;
  projectName: string;
  projectDescription: string;
  imageURL: string;
  createrDescription: string;
  projectUrl: string;
  githubRepoURL: string;
  githubURL: string;
  githubProfileURL: string;
  createrName: string;
  categories: any[];
  color:string;
}
```

Reply:
맞아용 근데 사실 json에 컬러코드도 추가를 덜 해놔서 .. ㅎㅎ
차근차근 해 놓을게요